.github/workflows/pr-checklist.yml

```
@@ -8,4 +8,4 @@ jobs:
   pr-checklist:
     runs-on: ubuntu-latest
     steps:
-      - uses: it-at-m/lhm_actions/action-templates/actions/action-pr-checklist@175e2b8e7162b91368e37f1ea5b74bf5ec3ee10a # v1.0.4
\ No newline at end of file
+      - uses: it-at-m/lhm_actions/action-templates/actions/action-pr-checklist@6b19f8dd2bee160bee82bde4fa04ac9f6c1d5e1e # v1.0.5
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)